config/routes.rb

```diff
@@ -2,11 +2,18 @@
   root 'home#index'
 
   resources :cards
-  put 'review' => 'trainer#review'
-
   resources :user_sessions, only: [:new, :create, :destroy]
   resources :users, only: [:new, :create, :destroy]
 
+  resources :blocks do
+    member do
+      put 'set_as_current'
+      put 'reset_as_current'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.